scmail.py

```diff
@@ -223,28 +254,48 @@ def export_key(ctx, fingerprint, is_file, is_pvt):
 
     pub, pvt = ctx.parent.gpg.export_key(fingerprint, export_pvt=is_pvt, password=passphrase)
 
+    if pub == '':
+        logging.error('The key not exists, please check fingerprint.')
+        logging.error('Mail Client Exits.')
+        return
+
     if is_file is False:
         ctx.parent.pp.pprint(pub)
         if is_pvt is True:
             ctx.parent.pp.pprint(pvt)
     else:
         with file_name.open('w') as f:
             f.write(pub)
-            f.write(pvt)
+            if pvt:
+                f.write(pvt)
 
     logging.info('Export key successful.')
 
 
 @click.command()
-@click.option('--file-path', '-p', prompt='Enter file path', required=True, help='The path of imported key.')
+@click.option('--file-path', '-p', prompt='Enter file path', required=True,
+              type=str, help='The path of imported key.')
+@click.option('--fingerprint', '-f', prompt='Enter fingerprint', required=True,
+              type=str, help='The fingerprint of this key/key pair.')
 @click.pass_context
-def import_key(ctx, file_path):
+def import_key(ctx, file_path, fingerprint):
+    # scaning key
+    key_data = Path(file_path)
+    res = ctx.parent.gpg.scan_file(file_path)
+
+    if res == []:
+        logging.error('The file path is wrong. File not found. Or No key (pair) in this file.')
+        return
+
+    if (fingerprint != res[0].get('fingerprint')):
```

Comment:
You can drop the parens here:
```suggestion
    if fingerprint != res[0].get('fingerprint'):
```

Reply:
Thanks, I already changed it.